lib/src/mt_dhcp.h

```diff
@@ -0,0 +1,47 @@
+/* SPDX-License-Identifier: BSD-3-Clause
+ * Copyright(c) 2023 Intel Corporation
+ */
+
+#ifndef _MT_LIB_DHCP_HEAD_H_
+#define _MT_LIB_DHCP_HEAD_H_
+
+#include "mt_main.h"
+
+enum mt_dhcp_udp_ports {
+  MT_DHCP_UDP_SERVER_PORT = 67,
+  MT_DHCP_UDP_CLIENT_PORT,
+};
+
+struct mt_dhcp_hdr {
```

Comment:
The struct is from some rfc, right? If yes, can you comment the RFC number here?